students/jarosz_natalia/lesson_08/regex_version_of strip.py

```diff
@@ -0,0 +1,14 @@
+import re
+
+
+def regex_strip(string, chars_to_remove):
```

Comment:
You should set the default value for ```chars_to_remove``` because Python strip function by default removes leading and trailing spaces from a string, so it should look like:

```def regex_strip(string, chars_to_remove=' '):```